docs/en/manuals/2dgraphics.md

```diff
@@ -5,5 +5,10 @@ brief: This manual is outdated
 
 # 2D Graphics
 
-This manual has been replaced by the [Graphics overview manual](/manuals/graphics).
+This manual has been replaced by the [Graphics overview manual](/docs/en/manuals/graphics).
```

Comment:
This is not a valid link. There is no page on https://defold.com/docs/en/manuals/graphics. Why did you decide to make this change? Did you encounter a broken link?